nodestream_akamai/akamai_utils/property_client.py

```diff
@@ -130,19 +208,19 @@ def describe_property_by_id(self, property_id: str) -> PropertyDescription:
             hostnames=list(hostnames),
         )
 
-    def describe_property_by_dict(self, property: dict) -> PropertyDescription:
+    def describe_property_by_dict(self, prop: dict) -> PropertyDescription:

```

Comment:
`property` is a python builtin, and I find it best to avoid using names that shadow them